src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,30 @@
+import { Component } from "react";
+
+interface ErrorBoundaryProps {
+  children: any;
+}
+
+interface ErrorBoundaryState {
+  hasError: boolean;
+}
+
+class ErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundaryState> {
+  constructor(props: ErrorBoundaryProps) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError() {
+    return { hasError: true };
+  }
```

Comment:
I did not manage to make this method run. How did you check if it is working?

Reply:
I didn't check it actually. Just tried to implement it according to docs. https://reactjs.org/docs/error-boundaries.html#introducing-error-boundaries.

Reply:
I'll approve only when I check it :)

Reply:
nvm , I saw it here: https://github.com/fedaipaca/movies-client-reactjs/pull/2#discussion_r1017069673

Reply:
👍